From 6db74c5dffcd925b477afeaaf9fc5e67ee24c0cf Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Tue, 7 Jan 2014 13:52:12 +0800
Subject: [PATCH 282/304] ENGR00293843 net:revert old commit to not add
 tcpdump

Not add tcpdump by default in user build, for it is root UID which will
break one cts-verifier item.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx6/imx6.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 77202e0..1166a8f 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -33,7 +33,6 @@ PRODUCT_PACKAGES += \
 	libRS					\
 	librs_jni				\
 	pppd					\
-	tcpdump					\
 	chat					\
 	ip-up-vpn				\
 	ip-up-ppp0				\
-- 
1.8.0

